DocumentFormat.OpenXml/DocumentFormat.OpenXml.csproj

```diff
@@ -249,6 +249,11 @@
       <Name>System.IO.Packaging</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="src\GeneratedCode\O12SchemaConstraintDatas.bin" />
+    <None Include="src\GeneratedCode\O14SchemaConstraintDatas.bin" />
+    <None Include="src\GeneratedCode\O15SchemaConstraintDatas.bin" />
```

Comment:
Those files were added when I corrected the ValidationResources.resx, which used all lowercase file names. That lead to a build error on Travis CI.